### PR TITLE
Created testSetName

### PR DESCRIPTION
**Pull request by ChaseMartinCRF01**

### CODE REVIEW

**Comment by ChaseMartinCRF01** [comment]
Added testCompareTo()

File: src/test/java/edu/kcc/animal/AnimalTest.java [hunk 1]
```diff
@@ -287,7 +291,12 @@ public void testToString() {
     // TODO
     @Test
     public void testCompareTo() {
-        fail("The test case is a prototype.");
+        setUp();
+        Animal other = null;
+        Animal animal = null;
+        int expResult = 0;
+        int result = animal.compareTo(other);
```
Comment (mlhaus) [inline src/test/java/edu/kcc/animal/AnimalTest.java:298]: You're comparing null to null. You need to compare two Animal objects with data.

**Review by mlhaus — CHANGES_REQUESTED** [review]
Don't compare null to null.

File: src/test/java/edu/kcc/animal/AnimalTest.java [hunk 2]
```diff
@@ -22,25 +24,35 @@
  * @author k0519415
  */
 public class AnimalTest {
-    
+
     private Animal animal;
-    
+    private static final String GOOD_NAME = "Snowball";
+    private static final String GOOD_ID = "0";
+    private static final String GOOD_SPECIES = "Artiodactyla ";
```
Comment (mlhaus) [inline src/test/java/edu/kcc/animal/AnimalTest.java:31]: I mentioned that we're only doing dogs and cats.

**Pull request merged**